### PR TITLE
Avillela python examples for docs

### PR DESCRIPTION
**Pull request by avillela**
Updated Python OTel examples

### CODE REVIEW

**Review by ocelotl — APPROVED** [review]
Just minor comments

File: python/launcher/server.py [hunk 1]
```diff
@@ -86,4 +98,4 @@ def sqlalchemy_integration(length):
 
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0")
+    app.run(host="0.0.0.0", port=8081, debug=True, use_reloader=False)
```
[the remark this answers is not in the document]
Comment (avillela) [reply python/launcher/server.py:100]: Not sure I know what you mean by that?

File: .gitignore [hunk 2]
```diff
@@ -7,12 +7,14 @@ package-lock.json
...
 .swp
 config/collector-config.yaml
 *scratchpad*
-venv/
-*.pyc
\ No newline at end of file
+venv*/
+*.pyc
+__pycache__
+*.bak*
```
[the remark this answers is not in the document]
Comment (avillela) [reply .gitignore:20]: I wanted to keep backups of my configs and I didn't want them to be committed to version control

File: python/opentelemetry/manual_instrumentation/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
+opentelemetry-api
+opentelemetry-sdk
+opentelemetry-exporter-otlp-proto-grpc
+opentelemetry-instrumentation-flask
+opentelemetry-instrumentation-requests
+
+# opentelemetry-api
```
[the remark this answers is not in the document]
Comment (avillela) [reply python/opentelemetry/manual_instrumentation/requirements.txt:7]: I'm still working on the manual instrumentation example.

**Pull request merged**